.github/workflows/ci.yml

```diff
@@ -364,33 +364,37 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        modules:
-          - ":trino-main"
-          - ":trino-tests"
-          - ":trino-raptor-legacy"
-          - ":trino-accumulo"
-          - ":trino-cassandra"
-          - ":trino-clickhouse"
-          - ":trino-hive"
-          - ":trino-hive -P test-parquet"
-          - ":trino-hive -P test-failure-recovery"
-          - ":trino-hive -P test-fault-tolerant-execution"
-          - ":trino-elasticsearch,:trino-elasticsearch -P test-stats"
-          - ":trino-mongodb"
-          - ":trino-kafka"
-          - ":trino-pinot"
-          - ":trino-redis"
-          - ":trino-mysql"
-          - ":trino-postgresql"
-          - ":trino-sqlserver"
-          - ":trino-memsql"
-          - ":trino-oracle"
-          - ":trino-kudu"
-          - ":trino-druid"
-          - ":trino-iceberg"
-          - ":trino-phoenix,:trino-phoenix5"
-          - ":trino-jdbc,:trino-base-jdbc,:trino-thrift,:trino-memory"
-          - ":trino-bigquery"
+        include:
```

Comment:
This new way is harder to read, since you can't easily visually scan the list of modules. It's almost worse that `modules` and `options` are the same length. If we changed `options` to `args`, that would help visually.

But I think we can improve using the inline map syntax, since having options is an exception. Also, we can drop the leading `:` from the module name and add below.

```yaml
include:
  - { modules: trino-main }
  - { modules: trino-hive }
  - { modules: trino-hive, options: "-P test-parquet" }
  - { modules: [trino-phoenix, trino-phoenix5] }
```
We could also change options to be a profile, which is more declarative, since we don't need generic options:
```yaml
  - { modules: trino-main }
  - { modules: trino-hive }
  - { modules: trino-hive, profile: test-parquet }
```
This should work to expand:
```yaml
run: $MAVEN test ${MAVEN_TEST} -pl :${{ join(matrix.modules, ':,') }} ${{ matrix.profile != "" && format('-P {0}', matrix.profile) }}

Reply:
Fully agree! I already had this issue when rebasing over your recent changes to this list :-) Applied all the suggestions.